arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java

```diff
@@ -118,6 +134,11 @@ private boolean shouldUseIdOfInstanceToMockResult(MockCategoryType category) {
     return !category.isEntryPoint() && !category.isSkipComparison();
   }
 
+  private boolean shouldBuildRecordOperationKey(Mocker mocker) {
+    return StringUtils.isEmpty(notUseSimilarityStrategyAppIds)
+        || !notUseSimilarityStrategyAppIds.contains(mocker.getAppId());
```

Comment:
You can split the notUseSimilarityStrategyAppIds into a string array and then check if the appid exists.
eg: notUseSimilarityStrategyAppIds is "10001234,10001235"    mocker.getAppId() is "10001"
this function will return true. It's not as expected.